extensions/amp-story/amp-story-analytics.md

```diff
@@ -104,6 +108,18 @@ The user's progress through the story, as a decimal in the range [0...1]. This r
 
 A boolean representing whether the story was muted when the accompanying trigger was fired.
 
+### `storyBookendComponentPosition`
+
+A number representing the position of the bookend component that the user clicked when the [accompanying trigger](#bookend-click-trigger-"on":-"story-bookend-click") was fired.
```

Comment:
The index? Position is a bit unclear as it could be a position in pixels, for example

Reply:
I would update to:

A number representing the position of the bookend component that the user clicked when the accompanying trigger, [`story-bookend-click`](#bookend-click-trigger-"on":-"story-bookend-click") was fired.

Reply:
Tried two address both comments. LMK what you think :)